### PR TITLE
Create Something_button.html

### PR DESCRIPTION
**Pull request by Rakesh-116**

### CODE REVIEW

File: modules/buttons/Something_button.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        border-width: 0;
+        border-radius:5px;
+        color:#ffffff;
+        font-weight:500;
+        font-size:30px;
+      }
+    </style>
+    <div>
+        <button class="danger-button">
+            <img src="https://res.cloudinary.com/dbcowhqno/image/upload/v1681525774/DPzTPPKeBxFpdAVnKxIg_-transformed-removebg-preview_crqt3h.png" style="height:25px;width:25px;"/>Danger
```
Comment (NtemKenyor) [inline modules/buttons/Something_button.html]: what if this sever/url goes down some day? Let's build a powerful and highly efficient system. Use svg for the icon. check recent buttons.

**Review by NtemKenyor — CHANGES_REQUESTED** [review]

**Comment by Rakesh-116** [comment]
Hey I have changed the code according to your request, Check it out

**Comment by Rakesh-116** [comment]
First of all sorry, I have done 6 more projects which are not assigned to
me, actually really did'nt know about it, sorry for that

**Comment by NtemKenyor** [comment]
you don't really need to apologize for that. I see it like an extra desire
to improve/learn....

I like your spirit.😊

**Comment by NtemKenyor** [comment]
However, I would advise that you focus on one issue and get your PR
accepted before you solve others.

It's easier that way...

**Comment by Rakesh-116** [comment]
will you accept my pull requests

**Pull request closed without merging**